.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
 		"devicepreferences",
 		"deviceprofiles",
 		"installedapps"
-	]
+	],
+	"typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Good config 👍. I'm assuming this gives us Intellisense for the version we are actually compiling with.

Reply:
I think VSCode added this when I changed the version to use the "workspace version". I have a newer version that was giving me more errors that I wasn't ready to fix yet.